### ISSUE TITLE
Configuration schema: both `typeshedPath` and `typingsPath` have the same id 

### ISSUE THREAD

**Issue by MikhailArkhipov** [microsoft/pyright#420]
**Describe the bug**
It appears that configuration schema has a typo: both `typeshedPath` and `typingsPath` have the same id as `#/properties/typingsPath`.

**To Reproduce**
https://github.com/microsoft/pyright/blob/699ea88df5f7c6f300c127bf35a776d3c06083ff/client/schemas/pyrightconfig.schema.json#L58
```json
    "typeshedPath": {
      "$id": "#/properties/typingsPath",
      "type": "string",
      "title": "Path to directory containing typeshed type stub files",
      "default": "",
      "pattern": "^(.*)$"
    },
    "typingsPath": {
      "$id": "#/properties/typingsPath",
      "type": "string",
      "title": "Path to directory containing custom type stub files",
      "default": "",
      "examples": [
        "src/typestubs"
      ],
      "pattern": "^(.*)$"
    },```
```
**Expected behavior**
Should they be different?

**Comment by erictraut** [comment]
Good catch. Thanks!

**Comment by erictraut** [comment]
This is now fixed in version 1.1.10, which I just published.
